GoogleSignIn/Sources/Public/GoogleSignIn/GIDAuthentication.h

```diff
@@ -64,9 +64,10 @@ typedef void (^GIDAuthenticationAction)(GIDAuthentication *_Nullable authenticat
 /// Get a valid access token and a valid ID token, refreshing them first if they have expired or are
 /// about to expire.
 ///
-/// @param action A callback block that takes a `GIDAuthentication` or an error if the attempt to
-///     refresh tokens was unsuccessful.  The block will be called asynchronously on the main queue.
-- (void)doWithFreshTokens:(GIDAuthenticationAction)action;
+/// @param completion A completion block that takes a `GIDAuthentication` or an error if the attempt
+///     to refresh tokens was unsuccessful.  The block will be called asynchronously on the main
+///     queue.
+- (void)doWithFreshTokens:(GIDAuthenticationAction)completion;
```

Comment:
Does changing the variable name from `action` -> `completion` here trigger the async conversion?  My understanding is that conversion is selector driven.

Reply:
Yep, take a look at https://github.com/google/GoogleSignIn-iOS/pull/187/files#diff-47b5fba4ca251cba0d58e2e7bd57c2921e900575dca59d87414dc2db3b61b340R30

Reply:
Resolved as we are deferring testing this sort of thing to the sample app (or some API testing target to be defined in the future).